src/screens/Inventory/CreatePurchaseRequest.tsx

```diff
@@ -0,0 +1,107 @@
+import { Button as MaterialButton, Typography } from '@material-ui/core';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import PhotoLibraryIcon from '@material-ui/icons/PhotoLibrary';
+import moment from 'moment';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { RouteComponentProps, useHistory } from 'react-router-dom';
+import BaseScreen from '../../components/BaseComponents/BaseScreen';
+import BaseScrollView from '../../components/BaseComponents/BaseScrollView';
+import Button from '../../components/Button';
+import TextField from '../../components/TextField';
+import { createPurchaseRequest } from '../../lib/airtable/request';
+import { currentInventoryProductSelector, currentInventorySelector } from '../../lib/redux/inventoryData';
+import { EMPTY_PURCHASE_REQUEST } from '../../lib/redux/inventoryDataSlice';
+import { userIdSelector } from '../../lib/redux/userData';
+import { getInventoryLastUpdated } from '../../lib/utils/inventoryUtils';
+import InventoryInfo from './components/InventoryInfo';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    content: {
+      color: theme.palette.text.primary,
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    cameraButton: {
+      backgroundColor: '#F7F9FC',
+      height: 130,
+      border: `3.5px dashed ${theme.palette.divider}`,
+      radius: '6px',
+    },
+});
+
+interface CreatePurchaseRequestProps extends RouteComponentProps {
+  classes: { content: string; cameraButton: string; };
+}
+
+function CreatePurchaseRequest(props: CreatePurchaseRequestProps) {
+  const { classes } = props;
+  const history = useHistory();
+  const userId = useSelector(userIdSelector);
+  const inventory = useSelector(currentInventorySelector);
+  const product = useSelector(currentInventoryProductSelector);
+
+  const [amountPurchased, setAmountPurchased] = useState(0.0);
+  const [amountSpent, setAmountSpent] = useState(0.0);
+  const [notes, setNotes] = useState("");
+
+  const handleAmountPurchasedInput = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setAmountPurchased(parseFloat(event.target.value as string));
+  }
+
+  const handleAmountSpentInput = (event: React.ChangeEvent<{ value: unknown }>) => {
+    setAmountSpent(parseFloat(event.target.value as string));
```

Comment:
We should make sure to restrict these to numbers. I put in letters and it let me submit and the amonut became a `NaN`

![image](https://user-images.githubusercontent.com/35501399/112728668-985fc400-8ee5-11eb-985c-a087d648bc4d.png)

Reply:
I made it default to 0 if NaN temporarily, but tabling this for a later PR to address app-wide error states and input validation.